truffle/src/main/java/org/jruby/truffle/RubyEngine.java

```diff
@@ -35,6 +36,7 @@ public RubyEngine(RubyInstanceConfig instanceConfig) {
 
         engine = PolyglotEngine.newBuilder()
                 .config(RubyLanguage.MIME_TYPE, INSTANCE_CONFIG_KEY, instanceConfig)
+                .config(RubyLanguage.MIME_TYPE, OptionsCatalogue.ARGUMENTS.getName(), instanceConfig.getArgv())
```

Comment:
We're getting rid of the `RubyInstanceConfig` on the line above when I convert all options over. So this is the only way we'll have to communicate arguments across. And it's a good thing because you'll then be able to set Ruby arguments for code that you invoke from other languages.